pkg/event/targetlist.go

```diff
@@ -17,14 +17,22 @@
 package event
 
 import (
+	"errors"
 	"fmt"
 	"sync"
+	"time"
 )
 
+const retryInterval = 3 // In Seconds
+
+// ErrNotConnected - indicates that the target connection is not active.
+var ErrNotConnected = errors.New("[Target] Client connection is not active")
```

Comment:
this error format is not recommended. `not connected to target server/service` makes sense.

Reply:
Changed :+1: